test/luajit-test-init.lua

```diff
@@ -1,9 +1,12 @@
 -- Disable strict for Tarantool.
 require("strict").off()
 
+local loaders = require('internal.loaders')
+
 -- XXX: lua-Harness test suite uses it's own tap.lua module
 -- that conflicts with the Tarantool's one.
 package.loaded.tap = nil
```

Comment:
Minor: I believe, that as a result of the corresponding patches removing builtin modules from `package.loaded`, this line can be removed, right?

Reply:
Technically, I guess, yes. However it would be fragile.

If there is no `require('tap')` call during tarantool's initialization, `package.loaded.tap` is `nil`. We only need `loaders.builtin.tap = nil` to choose an external module at next `require('tap')`.

However if some built-in module will depend on `tap` (and so call `require('tap')`), the built-in `tap` module will be written to `package.loaded.tap` and future require() will return it.

So the old `package.loaded.foo = nil` hack is now `package.loaded.foo = nil` plus `loaders.builtin.foo = nil`.

Reply:
Alright, let's leave it as is then.